File: src/main/java/ddp/api/security/PasswordHasher.java
package ddp.api.security;

/**
 * Manages the hashing of passwords
 */
public class PasswordHasher
{
    /**
     * Generates an one-way hash value for the password.
     * @param password Password for which to generate hashed value.
     * @return Hash value of the given password.
     */
    public static String getPasswordHash(String password)
    {
        //TODO DS check for null and empty string and hash password - encrupted hash needs to be the same in Java and C# - http://commons.apache.org/proper/commons-codec/
        return password;
    }
}

File: src/main/csharp/Ddp.Api/Security/PasswordHasher.cs
using System;

namespace Ddp.Api.Security
{
    public static class PasswordHasher
    {
        public static string GetPasswordHash(String password)
        {
            //TODO DS check for null and empty string and hash password - encrupted hash needs to be the same in Java and C#
            return password;
        }
    }
}